va/tlsalpn.go

```diff
@@ -198,6 +199,30 @@ func checkExpectedSAN(cert *x509.Certificate, name identifier.ACMEIdentifier) er
 	return nil
 }
 
+// Confirm that of the OIDs provided, all of them are in the provided list of
+// extensions. Also confirms that of the extensions provided that none are
+// repeated. Per RFC8737, allows unexpected extensions.
+func checkAcceptableExtensions(exts []pkix.Extension, requiredOIDs []asn1.ObjectIdentifier) error {
```

Comment:
Nit: this function takes the exts as input, the other checker above takes the whole cert as input. I'd prefer these have the same interface (but don't really care which one we standardize on).

Reply:
I think we might want to move this wholesale for use on CSRs, so let's keep it as-is?